Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -18,19 +18,52 @@
 #import "ABContactInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
+@interface __ABContactOperation : NSObject
+
+@property BOOL shouldDelete;
+@property WACContact* contact;
+
+- (id)initWithContact:(WACContact*)contact shouldDelete:(BOOL)shouldDelete;
+
+@end
+
+@implementation __ABContactOperation
+
+- (id)initWithContact:(WACContact*)contact shouldDelete:(BOOL)shouldDelete {
+    if (self = [super init]) {
+        self.shouldDelete = shouldDelete;
+        self.contact = contact;
+    }
+
+    return self;
+}
+
+@end
+
 @interface _ABAddressBookManager ()
 
+- (void)_setError:(CFErrorRef*)error message:(NSString*)message;
+
 @end
 
-@implementation _ABAddressBookManager
+@implementation _ABAddressBookManager {
+    NSMutableSet<WACContact*>* _toAdd;
+    NSMutableDictionary<NSString*, __ABContactOperation*>* _operations;
+    WACContactStore* _writableStore;
+    WACContactList* _contactList;
+}
 
 - (id)init {
     self = [super init];
     if (self) {
+        self->_toAdd = [[NSMutableSet alloc] init];
```

Comment:
Well, you use 'self->' in quite a few places. In my opinion, it should only be used when the name of the ivar and the name of what it's being set to are significantly similar.

---
In reply to: [76672822](https://github.com/Microsoft/WinObjC/pull/935#discussion_r76672822) [](ancestors = 76672822)